### PR TITLE
Added test for Configuration.getSpansArchiveTable

### PR DESCRIPTION
**Pull request by EinKrebs**

### CODE REVIEW

File: storage/config_test.go [hunk 1]
```diff
@@ -31,3 +33,15 @@ func TestSetDefaults(t *testing.T) {
 		})
 	}
 }
+
+func TestConfiguration_GetSpansArchiveTable(t *testing.T) {
+	const repetitionCount = 100
```
Comment (pavolloffay) [inline storage/config_test.go:38]: avoid using `const` for local variables. It's just noise.

File: storage/config_test.go [hunk 2]
```diff
@@ -31,3 +33,15 @@ func TestSetDefaults(t *testing.T) {
...
+	defaultConfig := Configuration{}
+	defaultConfig.setDefaults()
+	assert.Equal(t, defaultSpansTable+"_archive", defaultConfig.getSpansArchiveTable())
+	for i := 0; i < repetitionCount; i++ {
```
Comment (pavolloffay) [inline storage/config_test.go:42]: Why do we need this loop?

File: storage/config_test.go [hunk 3]
```diff
@@ -31,3 +33,15 @@ func TestSetDefaults(t *testing.T) {
...
+		tableName := "table_" + strconv.FormatUint(rand.Uint64(), 16)
```
Comment (pavolloffay) [inline storage/config_test.go:43]: avoid using random in tests. The tests should be repeatable and predictable

Comment (EinKrebs) [reply storage/config_test.go:43]: It's pseudo-random actually, so they are predictable. But yeah, I think I should get rid of random here.

**Review by pavolloffay — APPROVED** [review]

**Pull request merged**